packages/dev/core/src/Shaders/ShadersInclude/instancesDeclaration.fx

```diff
@@ -10,7 +10,7 @@
         uniform mat4 world;
     #endif
 #if defined(VELOCITY) || defined(PREPASS_VELOCITY) ||                  \
-            defined(PREPASS_VELOCITY_LINEAR)
+            defined(PREPASS_VELOCITY_LINEAR) || defined(VELOCITY_LINEAR)

```

Comment:
We don't support continuation line with `\`, you should put everything on the same line.